fastlane/Fastfile

```diff
@@ -447,28 +447,53 @@ platform :ios do
   lane :generate_docs do
     ENV["INCLUDE_DOCC_PLUGIN"] = "true"
     version_number = current_version_number
-    docs_folder = "docs/#{version_number}"
-    hosting_base_path = "purchases-ios/#{version_number}"
-
-    Dir.chdir("..") do
-      sh("swift",
-         "package",
-         "--disable-sandbox",
-         "--allow-writing-to-directory",
-         docs_folder,
-         "generate-documentation",
-         "--target",
-         "RevenueCat",
-         "--disable-indexing",
-         "--output-path",
-         docs_folder,
-         "--hosting-base-path",
-         "#{hosting_base_path}",
-         "--transform-for-static-hosting")
+    docs_repo_base_url = ENV["DOCS_REPO_BASE_URL"]
+    docs_repo_name = ENV["DOCS_REPO_NAME"]
```

Comment:
I also might add something like...

```rb
UI.user_error!("Missing environment variable: DOCS_REPO_BASE_URL") unless docs_repo_base_url
UI.user_error!("Missing environment variable: DOCS_REPO_NAME") unless docs_repo_name
``

Just to prevent some unknown error confusion if these aren't set 🤷‍♂️

Reply:
love it, adding